tests/labhub_test.py

```diff
@@ -72,8 +78,6 @@ def test_invite_cmd(self):
         self.inject_mocks('LabHub', mock_dict)
         testbot = self
 
-        plugins.labhub.os.environ['GH_TOKEN'] = 'patched?'
```

Comment:
note https://github.com/coala/corobo/pull/572#discussion_r206308989

You probably need a separate test method for this.
And that is not a newcomer problem --it is a test coverage problem which you need to sort out in your project.